projects/storefrontlib/src/cms-components/user/login-form/login-form.component.ts

```diff
@@ -51,20 +56,27 @@ export class LoginFormComponent implements OnInit, OnDestroy {
         this.winRef.nativeWindow.history.state;
 
       if (routeState && routeState['newUid'] && routeState['newUid'].length) {
-        this.prefillForm('userId', routeState['newUid']);
+        FormUtils.prefillForm(this.form, 'userId', routeState['newUid']);
       }
     }
   }
 
-  login(): void {
+  submitForm(): void {
```

Comment:
We do an explicit `submitForm` form the html, using the `submit` output emission. 
I'd prefer a pattern where we initialise the reactive form and add a form.submit handler in the controller. Simplifies the html, save us from a public controller method.

Reply:
the only solution I came across is indeed lighter for `html`, but super-complicated for `ts`, so I'm droping that idea for now

Reply:
my bad, this is actually the right way to do it!